pii_recognition/evaluation/character_level_evaluation.py

```diff
@@ -9,43 +9,94 @@
 from pii_recognition.labels.schema import Entity
 
 
+@dataclass
+class EntityPrecision:
+    entity: Entity
+    precision: float
+    entity_src: str = "predicted"
+
+
+@dataclass
+class EntityRecall:
+    entity: Entity
+    recall: float
+    entity_src: str = "ground_truth"
+
+
+@dataclass
+class TicketScore:
+    precisions: List[EntityPrecision]
```

Comment:
we should probably call this something my generic like `textscore` or `entityscore` ... i'm not sure of a good name, but the text doesn't have to be a Ticket :-D

Reply:
Agreed. I like the name `TextScore`, will change to this name then.